src/MethodScanner.cs

```diff
@@ -1,82 +1,90 @@
-// Copyright 2015 ThoughtWorks, Inc.
-//
```

Comment:
If the whole file is commented out, you may delete it and also the `IMethodScanner`.

Reply:
Removed